File: programmers/pg_lv1_이상한문자만들기.js
function solution(s) {
  let sArr = s.split(" ");
  let result = "";

  sArr.forEach((ele) => {
    let strArr = ele.split("");
    for (i = 0; i < strArr.length; i++) {
      if (i % 2 == 0) strArr[i] = strArr[i].toUpperCase();
      else strArr[i] = strArr[i].toLowerCase();
      result += strArr[i];
    }
    result += " ";
  });

  return result.slice(0, -1);
}

File: python/programmers/pg_lv1_이상한문자만들기.py
def solution(s):
    sArr = s.split(" ")
    result = ""
    
    for i in sArr:
        for (idx, j) in enumerate(i):
            if idx % 2 == 0: result += j.upper()
            else: result += j.lower()
                
        result += " "
    return result[:-1]